modules/ytp-assets-common/src/less/components/list--dataset.less

```diff
@@ -4,8 +4,15 @@
   padding: 0;
 }
 
+.dataset-item:last-of-type {
+  border: 1px solid #c9cdcf;
```

Comment:
Maybe add a new color-variable for this into `palette.less` (and use it here and couple lines below)